src/boot/vue-simple-acl.js

```diff
@@ -8,6 +8,7 @@ const userRoles = computed(() => ({ roles: userStore.roles }));
 const rules = () => defineAclRules((setRule) => {
   setRule('create-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagram'));
   setRule('create-diagram-from-template', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagramFromTemplate'));
+  setRule('create-component', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createComponent'));create-component rule in acl)
```

Comment:
problem : `user.roles.includes('CF_createComponent'));create-component rule in acl)`